### ISSUE TITLE
Direct Messaging

### ISSUE THREAD

**Issue by docapotamus** [pjuu/pjuu#107]
Allow users to message each other directly. This is very useful in some situations.

Should not be implemented like chat, and should not be a replacement to social interaction.

**Comment by deavmi** [comment]
If not by chat then how else?

**Comment by docapotamus** [comment]
Just sending a message. When I mean chat I mean live chat systems. I would prefer it to just be for passing around sensitive data. Very similar to DM on Twitter.

**Comment by sydneyjd** [comment]
To me allowing the users to message between themselves privately is a very important part of social media. I know that i won't be able to get quite a few people to join Pjuu just because they cannot message another person privately. 
Just some thoughts.

-Sydney

**Comment by deavmi** [comment]
Sounds good.

On 13 June 2016 3:29:51 PM SAST, sydneyjd <notifications@github.com> wrote:
>To me allowing the users to message between themselves privately is a
>very important part of social media. I know that i won't be able to get
>quite a few people to join Pjuu just because they cannot message
>another person privately. 
>Just some thoughts.
>
>-Sydney
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/pjuu/pjuu/issues/107#issuecomment-225580796

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by docapotamus** [comment]
I agree this is more important than I first thought.

We could allow users to set privacy constraints on who can send them DMs. Such as only users they 'Trust', users they are following, etc...

**Comment by deavmi** [comment]
Yes. That could make sense. obviously a way more smaller thread than the post layout.

**Comment by deavmi** [comment]
I also agree in that there needs to be Trust agree'd on both sides. So that person A must trust B and B must trust A in order for them to message each other.
